client/views/directory/DirectoryPage.js

```diff
@@ -17,10 +17,10 @@ function DirectoryPage() {
 	const [routeName] = useCurrentRoute();
 	const tab = useRouteParameter('tab');
 	const directoryRoute = useRoute('directory');
-
+	const navRoute = useRoute(routeName);
 	useEffect(() => {
 		if (routeName !== 'directory') {
-			return;
+			return navRoute.push();
```

Comment:
@Aman-Maheshwari  FYI: There's a warning being introduced here. Could you please resolve it? Thanks :smiley: 

![image](https://user-images.githubusercontent.com/34130764/141134191-4c253132-b016-43fc-98d5-ba7406945c3b.png)

Reply:
sure :+1:

Reply:
@murtaza98 can you please review it ? I have resolved the warning. Thank you :smile: